lib/bots/githubbot-types.d.ts

```diff
@@ -38,3 +39,9 @@ export interface GithubActionRegister extends GithubAction {
 
 // A GithubActionMethod is the method that will be used to process an event.
 export type GithubActionMethod = <T>(action: GithubAction, data: T) => Promise<void>;
+
+export interface GithubBotConfiguration extends ProcBotConfiguration {
+    githubbot: {
+        versionbot?: any;
```

Comment:
Similar to my below comment about `ProcBotConfiguration`, this should be the minimum interface we want to depend on at this stage, and adding `any` properties defeats that. I'd drop this for now.